dockside/__init__.py

```diff
@@ -1,9 +1,8 @@
-# -*- coding: utf-8 -*-
+from .nwis import Station
+from .io import *
+
+from .tests import test, teststrict, test_nowarnings
 
 __author__ = 'Lucas Nguyen'
-__version__ = '0.0.1'
+__version__ = '0.0.2'
```

Comment:
This is a major API change. I think we should be moving to a new major version?

Reply:
bumped to 0.1.0